src/playlist-selectors.js

```diff
@@ -103,17 +120,22 @@ export const comparePlaylistResolution = function(left, right) {
 
   rightWidth = rightWidth || window.Number.MAX_VALUE;
 
+  const leftBandwidth = getBandwidth(left, useAverageBandwidth);
+  const rightBandwidth = getBandwidth(right, useAverageBandwidth);
+
   // NOTE - Fallback to bandwidth sort as appropriate in cases where multiple renditions
-  // have the same media dimensions/ resolution
-  if (leftWidth === rightWidth &&
-      left.attributes.BANDWIDTH &&
-      right.attributes.BANDWIDTH) {
-    return left.attributes.BANDWIDTH - right.attributes.BANDWIDTH;
+  // have the same media dimensions/ resolution.
+  if (leftWidth === rightWidth && leftBandwidth && rightBandwidth) {
```

Comment:
I don't think we need to keep it for that reason. `comparePlaylistResolution` may be exposed, but its exposure is not documented anywhere as something that can/should be changed, so I think its reasonable to change it. This is the same reasoning as the call to not worry about this method (and the bandwidth comparator) being moved from `videojs-contrib-hls.js` to `playlist-selectors.js` such that `STANDARD_PLAYLIST_SELECTOR` no longer uses the comparators attached to the `Hls` object.